javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,166 @@
+
+    }
+    else if (temperature <= 30 && temperature > 20) {
+        return "Wear shorts and T-shirt";
+    }
+    else if (temperature <= 20 && temperature > 10) {
+        return "Wear summer jacket  ";
+    }
+    else if (temperature <= 10 && temperature > 0) {
+        return "Wear jacket and full size trouser";
+    }
+    else if (temperature <= 0 && temperature > -10) {
+        return "Wear winter jacket and trouser";
+    }
+    else if (temperature <= -10 && temperature > -20) {
+        return "Wear double winter jacket";
+    }
+    else if (temperature <= -20 && temperature > - 30) {
+        return "Wear very very warm clothes and stay inside";
+    }
+    else if (temperature <= -30) {
+        return "Are you at pluto? I can't suggest you any anything";
+    }
+    else {
+        return "Please Enter valid temprature";
+    }
+}
+
+let clothesToWear = recommendToWear(21);
+console.log(clothesToWear); // Logs out: "shorts and a t-shirt"
+
+/**addStudentToClass function */
+const class07Students = [];
+function addStudentToClass(studentName) {
+    const numberOfStudents = getNumberOfStudents()
+    /**check class has stil place for one student and student name is not Queen */
+    if (numberOfStudents >= 6 && studentName.toLowerCase() !== 'queen')
+        console.log('"Can not add more students to class 07"');
+    else if (studentName === '') {
+        console.log('Please Enter valid student name.');
+    }
+    else {
+        /*check student name is allready added or not*/
+        let isAllreadyAdded = false;
+        for (let i = 0; i < numberOfStudents; i++) {
+
+            if (class07Students[i].toLowerCase() === studentName.toLowerCase()) {
+                /**if student name found break the loop */
+                isAllreadyAdded = true; break;
+            }
+        }
+        if (isAllreadyAdded) {
+            console.log('Student ' + studentName + ' is already in the class');
+        }
+        else {/**(name of student is not already added) and (No of students are not more then 6) or( name is queen.) So, add new student   */
+            class07Students.push(studentName)
+
+        }
+    }
+
+}
+
+function getNumberOfStudents() {
+    return class07Students.length;
+
+}
+addStudentToClass('student1');
+addStudentToClass('');
+addStudentToClass('student2');
+addStudentToClass('student2');
+addStudentToClass('student3');
+addStudentToClass('student4');
+addStudentToClass('student5');
+addStudentToClass('student6');
+addStudentToClass('student7');
+addStudentToClass('queen');
+
+console.log(getNumberOfStudents())
+
+/**Candy helper optional */
+let boughtCandyPrices = [];
+let candyTypeList = ['Sweet', 'Chocolate', 'Toffee', 'Chewing-gum']
+let candyPricePerGram = [0.5, 0.7, 1.1, 0.03]
+
+function addCandy(candyType, weight) {
+    for (let i = 0; i < candyTypeList.length; i++) {
+        if (candyTypeList[i].toLocaleLowerCase() === candyType.toLocaleLowerCase()) {
```

Comment:
maybe it is more readable with a switch

Reply:
yes maybe but if we want add more  candy types then have to change in switch also, just in case candy types are 100  more

Reply:
right